capstone/src/test/java/com/google/step/search/ElasticsearchClientTest.java

```diff
@@ -71,4 +88,96 @@ public LowLevelHttpResponse execute() {
     testClient.updateRestaurantHeader(HEADER_1);
```

Comment:
Yeah, you make a good point. I'll look into that.

Reply:
After a bit of searching, it seems that this functionality is not offered by `MockHttpTransport`, unfortunately. Could I perhaps do some indirect assertions that, if pass, imply the requests were made? Or should I look for an alternative solution?

Reply:
Although, this seems like a good place to use a spy. Since I have already defined the behavior in the anonymous class declaration, I could wrap my `MockHttpTransport` in a spy verify any behaviors that way.

Reply:
I ended up adding the spy. Let me know if you think I should do something different.